### ISSUE TITLE
Create a tag for Syngine release

### ISSUE THREAD

**Issue by CTrabant** [krischer/instaseis#30]
Please create a tag at a stable point for syngine release.  Complete code freeze, potential release scheduled for Wednesday (9 December)

**Comment by krischer** [comment]
I'm still working on the custom STF functionality. It turned out to be fairly tricky and I did not have a lot of time due to AGU preparations - I think I can finish it by tomorrow. Is that still on the menu for the initial release? Otherwise I would schedule it after the christmas break.

Did you test the finite source route with syngine? It has been done on the instaseis side for a while but I would like you to test it first before I tag a version. Or is it not part of the initial release?

If both are postponed for now I can tag once you let me know if the slip/sliprate work for all databases with the latest instaseis version.

**Comment by alexhutko** [comment]
Finite sources w syngine?  No we don't have that yet and are OK with postponing that for a later update.  Same for user uploaded custom STFs.  

The STF width parameter would be nice to include in the initial release if you if you're sure it's bug/problem free.  If not, then we'll just postpone that one as well.

**Comment by krischer** [comment]
The STF width parameter is directly tied to the custom STF. I have a version that's tested a bit and appears to do the right thing but I cannot guarantee its bug free by tomorrow so I propose to postpone it for a later update.

The syngine client that's part of Instaseis can be used for custom source time functions if user's require it in the meanwhile.

Alright...so let me know once you tested the latest instaseis version and I'll tag that.

**Comment by CTrabant** [comment]
Current version confirmed working.  Please tag.

**Comment by krischer** [comment]
without the distance header in the sac file? You did not answer that one yet.

**Comment by CTrabant** [comment]
No, we would like DIST and O (origin time) fields populated.

Waiting on #31 and #32.

**Comment by CTrabant** [comment]
Please tag now.

**Comment by krischer** [comment]
Tag 0.1 is available: https://github.com/krischer/instaseis/tree/0.1

**Comment by krischer** [comment]
Tag 0.1.1 is now also available: https://github.com/krischer/instaseis/tree/0.1.1

Sorry about that - one of the tests was failing as suddenly the version number only had three digits due to the previous tag...

0.1 and 0.1.1 are identical except that the one test is fixed in 0.1.1.

**Comment by CTrabant** [comment]
NP.  0.1.1 is a release version for Syngine and is running now.  Thanks!
